core/backend/src/test/IModelTransformerUtils.ts

```diff
@@ -1650,9 +1650,9 @@ export class IModelToTextFileExporter extends IModelExportHandler {
     const element: Element = this.exporter.sourceDb.elements.getElement(aspect.element.id);
     return 1 + this.getIndentLevelForElement(element);
   }
-  protected onExportSchema(schema: Schema): void {
+  protected async onExportSchema(schema: Schema): Promise<void> {
     this.writeLine(`[Schema] ${schema.name}`);
-    super.onExportSchema(schema);
+    await super.onExportSchema(schema);

```

Comment:
Our convention is to use `return` instead of `await` for the last line of a method.